custom/eamar.json

```
@@ -0,0 +1,10 @@
+{
+    "welcome":
+    {
+        "mp3":"eamar/in/"
+    },
+    "goodbye":
+    {
+        "mp3":"eamar/out/"
+    }
+}
```

Hello, if you only want a song for indoor you can remove the goodbye part

```suggestion
{
    "welcome":
    {
        "mp3":"eamar/in/"
    }
}
```